cmd/oras/repository/list.go

```diff
@@ -35,8 +35,9 @@ type repositoryOptions struct {
 func listCmd() *cobra.Command {
 	var opts repositoryOptions
 	cmd := &cobra.Command{
-		Use:   "list REGISTRY [flags]",
-		Short: "[Preview] List the repositories under the registry",
+		Use:     "list REGISTRY [flags]",
+		Aliases: []string{"l"},
+		Short:   "[Preview] List the repositories under the registry",
```

Comment:
Would have expected the alias to be ls. Never seen l used before.

Reply:
Well Noted.